.github/workflows/build-pr.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       contents: read
     strategy:
       matrix:
-        KUBERNETES_VERSION: ["1.24.6"]
+        KUBERNETES_VERSION: ["1.24.6", "1.25.8", "1.26.3"]
```

Comment:
Should we just stick 2 versions? I think with the GK versions, each additional K8 version adds 2 more e2e tests.

Reply:
yeah, that seems too many tests, I'll keep 1.25.8 and 1.26.3.